packages/miniflare/src/plugins/kv/index.ts

```diff
@@ -96,9 +99,23 @@ export const KV_PLUGIN: Plugin<
 	}) {
 		const persist = sharedOptions.kvPersist;
 		const namespaces = namespaceEntries(options.kvNamespaces);
+		const KV_OBJECT_SERVICE_NAME = addPersistPathSuffix(
+			`${KV_PLUGIN_NAME}:object`,
```

Comment:
I have changed the service name from `kv:ns` to `kv:object:{persistPath}` just to avoid conflict with the namespace service name (`kv:ns:{namespace}`)